src/Microsoft.ML.Recommend/IMIValueMapper.cs

```diff
@@ -0,0 +1,93 @@
+﻿//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Runtime.Recommend
```

Comment:
please use `Recommender` as the namespace.

Reply:
Ok.

---
In reply to: [226367351](https://github.com/dotnet/machinelearning/pull/1263#discussion_r226367351) [](ancestors = 226367351)